src/ape/contracts/base.py

```diff
@@ -391,7 +391,7 @@ def __len__(self):
 
     def query(
         self,
-        *columns: List[str],
+        *columns: str,
```

Comment:
TIL https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
This will remain the same as it currently is defined

Reply:
Yup, I had thought the proper way to specify it was `*args: List[type]` but `*args: type` makes more sense